rke/k8s_version_info.go

```diff
@@ -113,7 +113,7 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		// It also includes ingress-nginx 0.32.0
 		"v1.15.12-rancher2-4": {
 			MinRancherVersion: "2.4.6-rc0",
-			MinRKEVersion:     "1.1.4-rc0",
+			MinRKEVersion:     "1.1.5-rc0",
```

Comment:
we need to only change this for 2-5, because we already released RKE 1.1.4 with 2-4 version and might cause confusion for later (even tho RKE doesn't dynamically pull data, so technically it's ok)

Reply:
Yep, changed the wrong one. Should be fixed now.